### PR TITLE
Docs: Remove -ignore deprecated

### PR DESCRIPTION
**Pull request by eleijonmarck**

### CODE REVIEW

File: tools/generate-docs.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # Generate docs!
-go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs -ignore-deprecated
+go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
```
Comment (eleijonmarck) [inline tools/generate-docs.sh:4]: this is the meat of the change

File: internal/resources/cloud/resource_cloud_api_key.go [hunk 2]
```diff
@@ -18,6 +18,8 @@ func ResourceAPIKey() *schema.Resource {
 	return &schema.Resource{
 		Description: `Manages a single API key on the Grafana Cloud portal (on the organization level)
 * [API documentation](https://grafana.com/docs/grafana-cloud/reference/cloud-api/#api-keys)
+
+!> Deprecated: please use ` + "`grafana_cloud_stack_service_account`" + ` and ` + "`grafana_cloud_stack_service_account_token`" + ` instead, see [Migrate API keys to Grafana service accounts using Terraform](https://grafana.com/docs/grafana-cloud/reference/migrate-api-keys-to-service-accounts/) for more information.
```
Comment (julienduchesne) [inline internal/resources/cloud/resource_cloud_api_key.go]: This is the wrong resource. This is a Cloud API key, this is absolutely not deprecated. What we should have deprecated is the cloud_stack_api_key resource

**Review by julienduchesne — CHANGES_REQUESTED** [review]

File: internal/resources/cloud/resource_cloud_api_key.go (hunk 2 above)
Comment (eleijonmarck) [reply internal/resources/cloud/resource_cloud_api_key.go]: i think we have to revisit the previous PR then - https://github.com/grafana/terraform-provider-grafana/pull/872/files#diff-04961555cdd45e8547f47c42af9a867501a2b2ebf4004768ae8d8830d728f77eR28

as we deprecated the cloud_api_key and not the cloud_stack one

Comment (eleijonmarck) [reply internal/resources/cloud/resource_cloud_api_key.go]: il update this PR with these changes

Comment (eleijonmarck) [reply internal/resources/cloud/resource_cloud_api_key.go]: see 4db6be893ebc72486661b6c6f203d9efba7fd164

Comment (eleijonmarck) [reply internal/resources/cloud/resource_cloud_api_key.go]: does this mean we have to make a new release no?

Comment (julienduchesne) [reply internal/resources/cloud/resource_cloud_api_key.go]: Yes, I will

**Review by julienduchesne — APPROVED** [review]
LGTM!

**Review by julienduchesne — APPROVED** [review]

**Comment by sg-qwt** [comment]
@julienduchesne From the docs linked https://grafana.com/docs/grafana-cloud/developer-resources/api-reference/cloud-api/#api-keys it says: `Caution: Cloud API keys are now deprecated. Use [Cloud Access Policies](https://grafana.com/docs/grafana-cloud/developer-resources/api-reference/cloud-api/#access-policies-and-tokens) instead.` Does this mean this resource is also deprecated in favor of grafana_cloud_access_policy and grafana_cloud_access_policy_token?

**Pull request merged**